main/test/ca/uwaterloo/flix/language/feature/Test.Expression.PutChannel.flix

```diff
@@ -0,0 +1,12 @@
+/////////////////////////////////////////////////////////////////////////////
+// Put channel expressions                                                 //
+/////////////////////////////////////////////////////////////////////////////
+
+def putChannelInt(ch1: Channel[Int], x: Int): Channel[Int] =
+    let _ = ch1 <- x;
+    ch1
+
+def testPutChannelInt(): Bool =
+    let ch = channel Int;
+    let x = putChannelInt(ch, 2);
+    assertEq!(<- x, 2)
```

Comment:
Obviously many more tests are needed. 

This should be a priority to test your typer and interpreter.